File: api/node/EnumVMStatus.go
package node

type EnumVMStatus string

const (
	EnumVMStatusrunning   EnumVMStatus = "running"
	EnumVMStatushalted    EnumVMStatus = "halted"
	EnumVMStatuspaused    EnumVMStatus = "paused"
	EnumVMStatushalting   EnumVMStatus = "halting"
	EnumVMStatusmigrating EnumVMStatus = "migrating"
	EnumVMStatusdeploying EnumVMStatus = "deploying"
	EnumVMStatuserror     EnumVMStatus = "error"
	EnumVMStatusstarting  EnumVMStatus = "starting"
)

File: pyclient/zeroos/orchestrator/client/EnumVMStatus.py
from enum import Enum


class EnumVMStatus(Enum):
    deploying = "deploying"
    running = "running"
    halted = "halted"
    paused = "paused"
    halting = "halting"
    migrating = "migrating"
    starting = "starting"
    error = "error"
